app/services/decidim/vocdoni/vocdoni_client.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Vocdoni
+    class VocdoniClient
+      attr_reader :wallet, :api_endpoint_env
+
+      def initialize(wallet:, api_endpoint_env:)
+        @wallet = wallet
+        @api_endpoint_env = api_endpoint_env
+      end
+
+      def fetch_election(vocdoni_election_id)
+        fetch_from_api("/elections/#{vocdoni_election_id}")
+      end
+
+      private
+
+      def fetch_from_api(path)
+        url = "#{base_url}#{path}"
+        response = Faraday.get(url)
+
+        JSON.parse(response.body) if response.success? && response.body.present?
+      end
+
+      def base_url
+        Decidim::Vocdoni.api_endpoint_url(api_endpoint_env)
```

Comment:
Related to my other commend, we should take the env from the one already defined in Decidim::Vocdoni
```suggestion
        Decidim::Vocdoni.api_endpoint_url
```